game.py

```diff
@@ -1,6 +1,9 @@
+		
+#Main      
 import random
 import sys
 
+doorOpened = false  #Global variable for gmae. Not sure if you guy want to change this logic. Ok with whatever u decide!:)
```

Comment:
It would be cool if we can find a way to handle state like this. Not sure yet what might work best.